### PR TITLE
kcptun: import

### PR DESCRIPTION
**Pull request by liushuyu**

### CODE REVIEW

File: extra-network/kcptun/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+VERSION=20160816
```
Comment (MingcongBai) [inline extra-network/kcptun/autobuild/build]: You can just use `$PKGDIR` here instead of making another one...

Comment (MingcongBai) [inline extra-network/kcptun/autobuild/build]: `$PKGVER`, sorry...

**Comment by liushuyu** [comment]
Fixed

**Comment by MingcongBai** [comment]
All-righty, thanks.

**Pull request merged**